src/test/kotlin/Decorator/littlekai/base/IngredientDecoratorTest.kt

```diff
@@ -1,15 +1,54 @@
 package Decorator.littlekai.base
 
+import Decorator.littlekai.ingredients.Pork
+import Decorator.littlekai.ingredients.Tuna
+import Decorator.littlekai.noodles.EggNoodles
+import Decorator.littlekai.noodles.UdonNoodles
+import Decorator.littlekai.noodles.WheatNoodles
 import org.hamcrest.CoreMatchers.`is`
 import org.junit.Assert.assertThat
+import org.junit.Assert.assertTrue
 import org.junit.Test
 
 class IngredientDecoratorTest {
 
+  val baseNoodles = UdonNoodles()
+  val ingredient = Pork(baseNoodles)
+  val doubleIngredient = Tuna(ingredient)
+
+  @Test
+  fun `check if decorated element is still Noodles`() =
```

Comment:
I don't like this kind of test because you are missing the test context related with the test.

For example I need have a look another part of your test code for see ingredient initialization.